src/mmrelay/cli.py

```
@@ -1506,6 +1785,189 @@ def handle_auth_command(args: argparse.Namespace) -> int:
+            f"   Legacy directories found: {', '.join(str(d) for d in paths_info['legacy_sources'])}"
+        )
+        print("   Migration will scan all legacy roots for data to migrate")
+        print(
+            "   💡 Run 'mmrelay service migrate --dry-run' to see what would be moved"
+        )
+        print("   💡 Run 'mmrelay migrate' to migrate data to new structure")
+
+    return 0
+
+
+def handle_verify_migration_command(args: argparse.Namespace) -> int:
```

Ruff (0.14.14)

[warning] Unused function argument: `args`

(ARG001)